community/sinit/build

```diff
@@ -1,10 +1,23 @@
 #!/bin/sh -e
 
-make CFLAGS="$CFLAGS -static"
+mv config.h sinit/config.h
 
-install -Dm 755 sinit "$1/usr/bin/sinit"
+(
+    cd sinit
+    make CFLAGS="$CFLAGS -static" PREFIX=/usr DESTDIR="$1" install
+    ln -s /usr/bin/sinit "$1/usr/bin/init"
+)
 
-install -Dm 755 sinit.runit.boot "$1/etc/rc.d/sinit.runit.boot"
-install -Dm 755 sinit.getty.boot "$1/etc/rc.d/sinit.getty.boot"
-install -Dm 755 sinit.pre.shutdown "$1/etc/rc.d/sinit.pre.shutdown"
-install -Dm 755 sinit.post.shutdown "$1/etc/rc.d/sinit.post.shutdown"
+
+install -Dm755 poweroff "$1/usr/bin/poweroff"
+install -Dm755 reboot "$1/usr/bin/reboot"
+
+install -Dm644 sinit-launch-services.boot "$1/etc/rc.d/sinit-launch-services.boot"
+install -Dm644 sinit.post.shutdown "$1/etc/rc.d/sinit.post.shutdown"
+
+(
+    cd ubase
+    make halt
+
+    install -Dm755 halt "$1/usr/bin/halt"
+)
```

Comment:
Is halt needed? Our `rc.shutdown` script doesn't use it at all.

Reply:
sinit.post.shutdown requires it.

Reply:
It could be replaced with @a-schaefers 's idea of shutting down from /proc. But I don't know how safe that is.

Reply:
Shutting down from `/proc` requires SYSRQ to be enabled in your kernel. Not always going to be the case (isn't in my config).

Reply:
> Shutting down from `/proc` requires SYSRQ to be enabled in your kernel. Not always going to be the case (isn't in my config).

I do not believe this is correct,  echo "1" > /proc/sys/kernel/sysrq should enable SYSRQ features temporarily until the next reboot (in this case, the change is immediately lost and would be harmless)

Reply:
From my system (without SYSRQ):

```sh
# This is as root.
-> cd /proc/sys/kernel/
-> pwd
/proc/sys/kernel
-> echo "1" > /proc/sys/kernel/sysrq
sh: can't create /proc/sys/kernel/sysrq: nonexistent directory
```

Reply:
Also:

```sh
-> ls -ld /proc/sys/kernel/
dr-xr-xr-x    1 root     root             0 Feb  4 15:19 /proc/sys/kernel/
```

Reply:
bummer